govtool/backend/sql/list-proposals.sql

```diff
@@ -83,11 +90,16 @@ FROM
     JOIN tx AS creator_tx ON creator_tx.id = gov_action_proposal.tx_id
     JOIN block AS creator_block ON creator_block.id = creator_tx.block_id
     LEFT JOIN voting_anchor ON voting_anchor.id = gov_action_proposal.voting_anchor_id
+    LEFT JOIN param_proposal as proposal_params ON gov_action_proposal.param_proposal = proposal_params.id
     LEFT JOIN off_chain_vote_data ON off_chain_vote_data.voting_anchor_id = voting_anchor.id
     LEFT JOIN off_chain_vote_gov_action_data ON off_chain_vote_gov_action_data.off_chain_vote_data_id = off_chain_vote_data.id
     LEFT JOIN voting_procedure ON voting_procedure.gov_action_proposal_id = gov_action_proposal.id
-    LEFT JOIN LatestDrepDistr ldd ON ldd.hash_id = voting_procedure.drep_voter
-        AND ldd.rn = 1
+    LEFT JOIN LatestDrepDistr ldd_drep ON ldd_drep.hash_id = voting_procedure.drep_voter
+        AND ldd_drep.rn = 1
+    LEFT JOIN LatestDrepDistr ldd_pool ON ldd_pool.hash_id = voting_procedure.pool_voter
```

Comment:
is counting SPO and CC votes meant to be a part of this PR?

Reply:
Not on the UI side. Just added this while adding protocol params to the list proposals

Reply:
Okay cool